baseband/gsb/base.py

```diff
@@ -111,8 +111,7 @@ class GSBStreamBase(VLBIStreamBase):
     def __init__(self, fh_ts, fh_raw, header0, thread_ids=None,
                  nchan=None, bps=None, complex_data=None,
                  samples_per_frame=None, payloadsize=None,
-                 frames_per_second=None, sample_rate=None,
-                 squeeze=True):
+                 sample_rate=None, squeeze=True):
```

Comment:
`VLBIStreamReaderBase.__init__` sets default values only for `sample_rate` and `squeeze`, hence why `sample_rate` is the second last argument for all readers.  I realized the order in the docstrings isn't consistent with the code, and have fixed this for all formats.

For GSB, however, we discussed yesterday removing the ability to auto-calculate the frame rate, since timestamps from the rawdump headers may drift by up to 10 ms (according to the GSB documentation on phased timestamps...).  I've thus made `sample_rate` a required argument, and adjusted the docstring as well.

Reply:
I think having it at the end consistently is perhaps more important. Maybe after all use that GSB really can have only a few settings? Anyway, find to leave it near the end here.